index.spec.js

```diff
@@ -101,7 +166,7 @@ describe('event-to-promise', function () {
     var error = new Error()
 
     // Node requires at least one error listener.
-    emitter.on('error', function noop () {})
+    emitter.once('error', function noop () {})
```

Comment:
You mean a bug in the test, because the library does not emit any events.

But fair point.